Prosjektrapport-V2.md

```diff
@@ -27,6 +27,13 @@
 
 ### Interessenter og brukerbeskrivelser
 
+En interessent kan omfatte enkeltpersoner, team eller organisasjoner som er interesserte i eller som kan påvirke direkte eller indirekte av utviklingsprosessen og det endelige utfallet av prosjektet. Interessentene i HytteApp inkluderer ulike gruppe interessenter, hver med spesifikke forventninger og interesser i forhold til appens funksjonalitet, ytelse og daglig bruk. 
+
+Blant de direkte interessentene er hytteeierne, som primært bruker verktøyet for å registrere seg og bestille brøytetjenester. Dette viser tydelig viser deres ønske for tilgjengelighet og brukervennlighet. Deretter har vi brøytemannskapet, de operative interessentene, som er avhengige av dette appen for å koordinere og utføre sitt daglig arbeid. En annen direkte interessentgruppe er administrasjon og Hytteservice. Deres intensjon med HytteApp er å håndtere bestillinger og overvåke brøyteoperasjonene, samt hente ut rapporter og innsikt som kan brukes for å forbedre tjenesten.  
+
+En annen kategori ville inkludere de som har en indirekte innvirkning i HytteApp-prosjektet. Dette omfatter de som har ansvaret for sikre at forskrifter, standarder regelverk blir etterfulgt. Et eksempel på en indirekte aktør som likevel har stor betydning for appen, er Statens Kartverk. Deres rolle er å levere kritiske eiendomsdata som er nødvendige for nøyaktig lokalisering av hytter. Dette er essensielt for at brøyting kan planlegges og gjennomføres effektivt. En annen indirekte aktør er de regulatoriske myndighetene, som sørger for at HytteApp følger personvernlovgivingen GDPR og standarder for universell utforming. Dette er avgjørende for tjenestens tilgjengelighet og lovlighet. 
```

Comment:
...Dette omfatter de som har ansvaret for sikre at forskrifter, standarder og regelverk blir etterfulgt....